ml-distance-euclidean/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for ml-distance-euclidean v1.0.0
+// Project: https://github.com/mljs/distance-euclidean
+// Definitions by: Diana Caro <https://github.com/CaribeSoy/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'ml-distance-euclidean' {

```

Comment:
could you strip `declare module 'ml-distance-euclidean' {` ?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred